### PR TITLE
Cache Release Status result for 5 minutes

### PR DESCRIPTION
**Pull request by vstinner**

### CODE REVIEW

**Comment by vstinner** [comment]
@pablogsal @ambv: Military grade optimization, I reduced the cache to 5 minutes.

File: master/custom/templates/releasedashboard.html [hunk 1]
```diff
@@ -46,4 +46,12 @@ <h4 class="panel-title">
...
         </div>
     </div>
+    <div class="container">
+        <small>Generated at <time id="generatedAt" datetime="{{generated_at}}">{{generated_at}}</time></small>
+    </div>
 </div>
+<script type="javascript">
+    let elem = document.getElementById("generatedAt");
+    let date = new Date(elem.dateTime);
+    elem.innerText = date.toLocaleDateString("en-CA") + " " + date.toLocaleTimeString("pl-PL");
```
Comment (vstinner) [inline master/custom/templates/releasedashboard.html:56]:
```suggestion
    elem.innerText = date.toLocaleDateString("en-US") + " (UTC)";
```

File: master/custom/templates/releasedashboard.html [hunk 2]
```diff
@@ -46,4 +46,12 @@ <h4 class="panel-title">
             </div>
         </div>
     </div>
+    <div class="container">
+        <small>Generated at <time id="generatedAt" datetime="{{generated_at}}">{{generated_at}}</time></small>
```
Comment (vstinner) [inline master/custom/templates/releasedashboard.html:50]:
```suggestion
        <small>Generated at <time id="generatedAt" datetime="{{generated_at}}">{{generated_at}}</time> (cached for 5 minutes)</small>
```

Comment (vstinner) [reply master/custom/templates/releasedashboard.html:50]: Maybe the cache duration should be a template parameter.

Comment (ambv) [reply master/custom/templates/releasedashboard.html:50]: No, this won't be cached for 5 minutes.

File: master/custom/templates/releasedashboard.html (hunk 1 above)
Comment (ambv) [reply master/custom/templates/releasedashboard.html:56]: No, this is invalid.

File: master/custom/release_dashboard.py [hunk 3]
```diff
@@ -53,7 +58,28 @@ def main():
 
             failed_builds_by_branch[branch].append((builder, last_build))
 
+        generated_at = datetime.datetime.now(tz=datetime.UTC)
```
Comment (ambv) [inline master/custom/release_dashboard.py:61]:
```
datetime.UTC
Alias for the UTC timezone singleton [datetime.timezone.utc](dfile:///Users/ambv/Library/Application%20Support/Dash/DocSets/Python_3/Python%203.docset/Contents/Resources/Documents/doc/library/datetime.html#datetime.timezone.utc).

New in version 3.11.
```

**Comment by vstinner** [comment]
> @ambv: Use datetime.timezone.utc which is available on 3.9

@zware: Once you mentioned that you considered to upgrade Python on the buildbot? Is it still the case? :-)

File: master/custom/templates/releasedashboard.html (hunk 1 above)
Comment (vstinner) [reply master/custom/templates/releasedashboard.html:56]: Why is the date formatted twice, in 2 different locales?

**Comment by zware** [comment]
> @zware: Once you mentioned that you considered to upgrade Python on the buildbot? Is it still the case? :-)

I'm in favor, but have made no attempts towards it (that I remember :)).  Pretty sure it will need to be done via the [Salt configuration](https://github.com/python/psf-salt/blob/main/salt/buildbot/init.sls).

**Pull request merged**